Sources/Private/CoreAnimation/CoreAnimationLayer.swift

```diff
@@ -92,10 +95,16 @@ final class CoreAnimationLayer: BaseAnimationLayer {
     didSet { reloadImages() }
   }
 
+  /// The `AnimationTextProvider` that `TextLayer`'s use to retrieve texts,
+  /// that they should use to render their text context
+  var textProvider: AnimationTextProvider {
+    didSet { reloadTexts() }
```

Comment:
`reloadFonts()` / `reloadTexts()` doesn't cause `setupAnimations(context:)` to be called again. Instead I think we need to call `rebuildCurrentAnimation()`, like we do in `setValueProvider(_:keypath:)` below.

Reply:
Got you! Thanks! 
Fixed in [b542f66](https://github.com/airbnb/lottie-ios/pull/1723/commits/b542f66fc79f97f4d370a0f0e0876ff4750a3cd7) ✅